arangod/Graph/Providers/SingleServerProvider.cpp

```diff
@@ -159,6 +160,7 @@ auto SingleServerProvider<Step>::expand(
 
         callback(Step{id, std::move(eid), previous, step.getDepth() + 1,
                       _opts.weightEdge(step.getWeight(), edge), cursorID});
+        // TODO [GraphRefactor]: Why is cursorID set, but never used?
```

Comment:
There is one implementation that used, it, but there is a high probability we do not need it anymore after refactoring is complete.

Reply:
Will add this note to the code itself. Will then clean up during TraversalNode refactor.